salon_creator/lib/screens/user_profile_edit_screen.dart

```diff
@@ -0,0 +1,320 @@
+        _buildNameEditDisplay(userModel),
+        _buildAboutMeEditDisplay(userModel),
+      ],
+    ).toList());
+
+    if (_opInProgress) {
+      profileItems.insert(0, LinearProgressIndicator());
+    }
+
+    return WillPopScope(
+      child: Scaffold(
+        appBar: _buildAppBar(userModel),
+        body: Padding(
+          padding: const EdgeInsets.symmetric(horizontal: 16),
+          child: ListView(
+            children: profileItems,
+          ),
+        ),
+      ),
+      onWillPop: _onBackButtonPressed,
+    );
+  }
+
+  Widget _buildAppBar(UserModel userModel) {
+    return AppBar(
+      title: Text('プロフィールの編集'),
+      iconTheme: IconThemeData(color: Colors.black),
+      actions: [
+        TextButton(
+          onPressed: _profileUpdated
+              ? () async {
+                  await _saveProfile(userModel);
+                  Navigator.pop(context);
+                }
+              : null,
+          child: Text('終了'),
+          style: ButtonStyle(
+            foregroundColor: MaterialStateProperty.resolveWith((states) =>
+                states.any((element) => element == MaterialState.disabled)
+                    ? Colors.grey
+                    : primaryColor),
+          ),
+        ),
+      ],
+    );
+  }
+
+  Future<void> _saveProfile(UserModel userModel) async {
+    setState(() {
+      _opInProgress = true;
+    });
+
+    userModel.profile.name = nameController.text;
+    userModel.profile.aboutMeText = aboutMeController.text;
+    if (_imageFile != null) {
+      userModel.profile.imageUrl = await _uploadImageAndGetUrl(_imageFile);
+    }
+
+    await FirebaseFirestore.instance
+        .collection(constants.DBCollection.users)
+        .doc(FirebaseAuth.instance.currentUser.email)
+        .update(userModel.toMap());
+
+    setState(() {
+      _opInProgress = false;
+    });
+  }
+
+  Future<String> _uploadImageAndGetUrl(XFile imageFile) async {
+    var snapshot = await FirebaseStorage.instance
+        .ref()
+        .child('images')
+        .child('${Uuid().v1()}.png')
+        .putFile(File(imageFile.path));
+
+    return snapshot.ref.getDownloadURL();
+  }
+
+  Widget _buildNameEditDisplay(UserModel userModel) {
+    return Column(
+      crossAxisAlignment: CrossAxisAlignment.start,
+      children: [
+        SizedBox(height: 16),
+        Text(
+          '名前',
+          style: TextStyle(
+            fontSize: 20,
+            fontWeight: FontWeight.w700,
+            color: primaryColor,
+          ),
+        ),
+        SizedBox(height: 4),
+        _buildNameFormField(userModel),
+        SizedBox(height: 24),
+      ],
+    );
+  }
+
+  TextFormField _buildNameFormField(UserModel userModel) {
+    return TextFormField(
+      controller: nameController,
+      onChanged: (_) => _updateScreenContext(userModel),
+      decoration: InputDecoration(
+        border: OutlineInputBorder(
+          borderRadius: BorderRadius.circular(12.0),
```

Comment:
In my opinion, as border raidus 5.0 looks better than 12.0. 
```suggestion
          borderRadius: BorderRadius.circular(5.0),
```